File: 14.c
#include <stdio.h>
 
int compute_gcd(int x, int y) {
    while(y) {
        int temp = y;
        y = x % y;
        x = temp;
    }
  
    return x;
}
 
int compute_lcm(int x, int y) {
    int gcd = compute_gcd(x, y);
    int lcm = (x * y) / gcd;
  
    return lcm;
}
 
int main() {
    int num1, num2;
    scanf("%d %d", &num1, &num2);
    printf("%d\n", compute_lcm(num1, num2));

    return 0;
}

File: 14.py
def compute_gcd(x, y):
   while(y):
       x, y = y, x % y
   return x
 
def compute_lcm(x, y):
   lcm = (x*y)//compute_gcd(x,y)
   return lcm
 
n = input().split()
num1 = int(n[0])
num2 = int(n[1])
 
print(compute_lcm(num1, num2))
